trustedlogin-vendor.php

```diff
@@ -29,7 +29,7 @@
 //Set this to true, in wp-config.php to log all PHP errors/warnings/notices to trustedlogin.log
 // Code: define( 'TRUSTEDLOGIN_DEBUG', true );
 if( ! defined( 'TRUSTEDLOGIN_DEBUG') ){
-	define( 'TRUSTEDLOGIN_DEBUG', false );
+	define( 'TRUSTEDLOGIN_DEBUG', null );
```

Comment:
@zackkatz  I changed the default, so we can now if it is set to a boolean, or not used (null)